lib/takeaway.rb

```diff
@@ -0,0 +1,40 @@
+require 'dotenv'
+Dotenv.load
+
+require 'takeaway_dish'
+require 'takeaway_menu'
+require 'takeaway_order'
+require 'print_menu'
+require 'total_checker'
+require 'sms_sender'
+
+class Takeaway
+
+  attr_reader :takeaway_menu, :takeaway_order_class, :menu_printer, :total_checker, :sms_sender
+
+  def initialize(args)
+    @takeaway_menu = args[:takeaway_menu]
```

Comment:
I'd be tempted to make this default args in the constructor that can be overwritten by the tests.